CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## [0.2.2] - 2024-XX-XX
 
+**Documentation**
+
+- Various fixes in documentation, including documenting returned models.
+
+**Breaking changes**
+
+- The `return_posteriors` argument has been removed and replaced with `return_model`.
+  An instance of one of two previously internal classes, `ExpectationPropagationModel`
```

Comment:
Yeah, I was also a little uncomfortable about "model", but I want to be able to say

```python
ts, something = tsdate.date(ts, mu, return_something=True)
```

Have you got a better suggestion for "something". I guess "fit" might work actually?

Reply:
"return_fit" is fine, I think. I'd prefer to keep the class name as "ExpectationPropagation", no need to add a suffix.

Reply:
Good call. I completely see your point: "fit" is more appropriate and accurate than "model" (as well as it being shorter to type), so thanks for that suggestion.

The equivalent to the "ExpectationPropagation" class is now called "InsideOutside", but I'm not so happy with that name, partly because that fit object is returned from both the inside-outside and the outside-maximisation methods. Do you by any chance have a better suggestion for a name describing that non-variational approach?

Reply:
> Do you by any chance have a better suggestion for a name describing that non-variational approach?

BeliefPropagation?